apps/angular/permissions/src/app/commons/admin-auth.directive.ts

```diff
@@ -0,0 +1,32 @@
+import {
+  Directive,
+  Input,
+  OnInit,
+  TemplateRef,
+  ViewContainerRef,
+} from '@angular/core';
+import { UserStore } from '../user.store';
+
+@Directive({
+  selector: '[hasRoleSuperAdmin]',
+  standalone: true,
+})
+export class AdminAuthDirective implements OnInit {
+  @Input()
+  hasRoleSuperAdmin: boolean = false;
+
+  constructor(
+    private templateRef: TemplateRef<any>,
+    private viewContainer: ViewContainerRef,
+    private userStore: UserStore,
+  ) {}
+
+  ngOnInit() {
+    this.userStore.user$.subscribe({
```

Comment:
don't forget to unsuscribe

Reply:
Thank you, I won't next time. Fixed on next commit